app/forms/fields/year_date_field.py

```diff
@@ -26,11 +28,28 @@ def data(self):
 
 
 class YearDateField(FormField):
-    def __init__(self, validators, **kwargs):
+    def __init__(
+        self,
+        *,
+        validators: Sequence[Callable],
+        label: str | None = None,
+        description: str,
+        **kwargs: Any,
+    ):
         form_class = get_form_class(validators)
-        super().__init__(form_class, **kwargs)
-
-    def process(self, formdata, data=None, extra_filters=None):
+        super().__init__(
+            form_class,
+            label=label,
+            description=description,
+            **kwargs,
+        )
+
+    def process(
+        self,
+        formdata: MultiDict,
+        data: UnsetValue = unset_value,
+        extra_filters: None = None,
```

Comment:
Documentation suggests `extra_filters` is a list of filter functions so I think this one may be `Sequence[Callable] | None` similar to some others